### ISSUE TITLE
Federated Learning addon description does not say it's a Shield study

### ISSUE THREAD

**Issue by dranov** [florian/federated-learning-addon#15]
It is slightly unsettling that, when you are selected for the a Shield study, the respective add-on just pops up as an installed extension. For federated learning, I thought the extension might be some kind of malware and would have deleted it if I hadn't figured out it was a Shield extension by searching for "federated" in `about:config`.

![fl-config](https://user-images.githubusercontent.com/1622998/43675138-14286320-97d7-11e8-8133-ea4586e4bccc.PNG)

This might be improved if either **a)** the description of the add-on mentioned Shield and reminded the user that they signed up for it, or **b)** shield extensions would be included in a separate category in `about:addons`, e.g. "Shield studies", rather than "Extensions".

![fl-addon](https://user-images.githubusercontent.com/1622998/43675137-13ebff2a-97d7-11e8-91fa-eacc27b3352a.PNG)

**Comment by florian** [comment]
I agree that this system for shield studies is far from optimal. In terms of having the study show up as an addon, there is little I can do from my side.

But you're definitely right that it would be a good idea to mention in the description that this is a shield study. Of course, malware could also just add something like that to descriptions, so a better solution would still work on a Firefox level.

**Comment by motin** [comment]
Quick update: Study add-ons should use `"hidden": true` in their manifest, so that they no longer show up in about:addons. They will still show up in about:studies as they are deployed by Normandy.
